src/test/java/com/stripe/service/ExternalAccountServiceTest.java

```diff
@@ -0,0 +1,145 @@
+package com.stripe.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.BankAccount;
+import com.stripe.model.Card;
+import com.stripe.model.ExternalAccount;
+import com.stripe.model.ExternalAccountCollection;
+import com.stripe.net.ApiResource;
+import com.stripe.param.ExternalAccountCreateParams;
+import com.stripe.param.ExternalAccountListParams;
+import com.stripe.param.ExternalAccountRetrieveParams;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+class ExternalAccountServiceTest extends BaseStripeTest {
+  public static final String ACCOUNT_ID = "acct_123";
+  public static final String BANK_ACCOUNT_ID = "btok_123";
+  public static final String CARD_ID = "card_123";
+
+  private ExternalAccountService externalAccountService = new ExternalAccountService();
```

Comment:
Currently, static methods on Card/BankAccount: list/retrieve/create are done on the polymorphic collection.
`ExternalAccountCollection` and `PaymentSourceCollection`.
We can consider having similar `ExternalAccountService` and `PaymentSourceService`,
or we can move `list/retrieve/create` to Card/BankAccount and similarly rely on params overloading